src/networking.c

```diff
@@ -2116,6 +2117,48 @@ void resetClient(client *c) {
     }
 }
 
+/* Initializes the shared query buffer to a new sds with the default capacity */
+void initSharedQueryBuf(void) {
+    shared_qb = sdsnewlen(NULL, PROTO_IOBUF_LEN);
```

Comment:
I see more thread-local-storage variables being introduced into the code base recently. I would suggest naming them with a common prefix, for instance `thread_shared_qb` or `thread_local_shared_qb`, for readability.

```suggestion
    thread_shared_qb = sdsnewlen(NULL, PROTO_IOBUF_LEN);
```

Reply:
Thanks, changed per your suggestion.